po/pt_BR.po

```diff
@@ -1,7 +1,7 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR THE paru'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the paru package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Juliano Marcelino (ojmarcelino), 2021.
```

Comment:
here you can place `Pedro Liberatti <ferrahwolfeh@protonmail.com>, 2021`

and place your name in the last translator field